server/mscalendar/notification.go

```diff
@@ -248,34 +259,70 @@ func (processor *notificationProcessor) updatedEventSlackAttachment(n *remote.No
 		return false, nil
 	}
 
+	allChanges := append([]string{}, added...)
+	allChanges = append(allChanges, updated...)
+	allChanges = append(allChanges, deleted...)
```

Comment:
Just a suggestion, up to you if you want to make a change here. It's an extra line but I prefer the first one here:

```suggestion
	var allChanges []string
	allChanges = append(allChanges, added...)
	allChanges = append(allChanges, updated...)
	allChanges = append(allChanges, deleted...)
```

or

```suggestion
	allChanges := append(added, updated...)
	allChanges = append(allChanges, deleted...)
```